isis/cmake/BuildDocs.cmake

```diff
@@ -464,8 +472,10 @@ endfunction(build_object_docs)
 function(build_docs)
 
   message("Building Isis Documentation...")
+  message(" Version is: " ${VERSION})
 
   # Set up output directory and a temporary directory for building
+  set(docVersion       7.0.0)
```

Comment:
Ignoring, but noting (for later reviews)

Reply:
Same as above, just set this to `${CMAKE_PROJECT_VERSION}` and do it prior to the message above so the message can just use `${docVersion}`